server/repository/src/db_diesel/invoice_line_row.rs

```diff
@@ -97,6 +98,7 @@ pub struct InvoiceLineRow {
     pub foreign_currency_price_before_tax: Option<f64>,
     pub item_variant_id: Option<String>,
     pub linked_invoice_id: Option<String>,
+    pub donor_default_id: Option<String>,
```

Comment:
Have left the same naming convention as per OG. I thought changing to 'donor_id' would add confusion unnecessarily